java.lang.StackOverflowError on HeapAnalyzerService

After navigating a while through my app I got a StackOverFlowError...

```
        AndroidRuntime  E  FATAL EXCEPTION: IntentService[HeapAnalyzerService]
                        E  Process: com.weheartit:leakcanary, PID: 14442
                        E  java.lang.StackOverflowError
                        E      at java.io.BufferedOutputStream.write(BufferedOutputStream.java:135)
                        E      at java.io.DataOutputStream.writeInt(DataOutputStream.java:180)
                        E      at java.io.ObjectOutputStream.writeCyclicReference(ObjectOutputStream.java:812)
                        E      at java.io.ObjectOutputStream.dumpCycle(ObjectOutputStream.java:427)
                        E      at java.io.ObjectOutputStream.writeClassDesc(ObjectOutputStream.java:754)
                        E      at java.io.ObjectOutputStream.writeNewObject(ObjectOutputStream.java:1369)
                        E      at java.io.ObjectOutputStream.writeObjectInternal(ObjectOutputStream.java:1671)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1517)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1481)
                        E      at java.io.ObjectOutputStream.writeNewArray(ObjectOutputStream.java:1205)
                        E      at java.io.ObjectOutputStream.writeObjectInternal(ObjectOutputStream.java:1662)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1517)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1481)
                        E      at java.io.ObjectOutputStream.writeFieldValues(ObjectOutputStream.java:979)
                        E      at java.io.ObjectOutputStream.defaultWriteObject(ObjectOutputStream.java:368)
                        E      at java.io.ObjectOutputStream.writeHierarchy(ObjectOutputStream.java:1074)
                        E      at java.io.ObjectOutputStream.writeNewObject(ObjectOutputStream.java:1404)
                        E      at java.io.ObjectOutputStream.writeObjectInternal(ObjectOutputStream.java:1671)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1517)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1481)
                        E      at java.util.ArrayList.writeObject(ArrayList.java:648)
                        E      at java.lang.reflect.Method.invokeNative(Native Method)
                        E      at java.lang.reflect.Method.invoke(Method.java:515)
                        E      at java.io.ObjectOutputStream.writeHierarchy(ObjectOutputStream.java:1053)
                        E      at java.io.ObjectOutputStream.writeNewObject(ObjectOutputStream.java:1404)
                        E      at java.io.ObjectOutputStream.writeObjectInternal(ObjectOutputStream.java:1671)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1517)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1481)
                        E      at java.io.ObjectOutputStream.writeFieldValues(ObjectOutputStream.java:979)
                        E      at java.io.ObjectOutputStream.defaultWriteObject(ObjectOutputStream.java:368)
                        E      at java.io.ObjectOutputStream.writeHierarchy(ObjectOutputStream.java:1074)
                        E      at java.io.ObjectOutputStream.writeNewObject(ObjectOutputStream.java:1404)
                        E      at java.io.ObjectOutputStream.writeObjectInternal(ObjectOutputStream.java:1671)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1517)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1481)
                        E      at java.util.ArrayList.writeObject(ArrayList.java:648)
                        E      at java.lang.reflect.Method.invokeNative(Native Method)
                        E      at java.lang.reflect.Method.invoke(Method.java:515)
                        E      at java.io.ObjectOutputStream.writeHierarchy(ObjectOutputStream.java:1053)
                        E      at java.io.ObjectOutputStream.writeNewObject(ObjectOutputStream.java:1404)
                        E      at java.io.ObjectOutputStream.writeObjectInternal(ObjectOutputStream.java:1671)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1517)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1481)
                        E      at java.io.ObjectOutputStream.writeFieldValues(ObjectOutputStream.java:979)
                        E      at java.io.ObjectOutputStream.defaultWriteObject(ObjectOutputStream.java:368)
                        E      at java.io.ObjectOutputStream.writeHierarchy(ObjectOutputStream.java:1074)
                        E      at java.io.ObjectOutputStream.writeNewObject(ObjectOutputStream.java:1404)
                        E      at java.io.ObjectOutputStream.writeObjectInternal(ObjectOutputStream.java:1671)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1517)
                        E      at java.io.ObjectOutputStream.writeObject(ObjectOutputStream.java:1481)
                        E      at java.util.ArrayList.writeObject(ArrayList.java:648)
                        E      at java.lang.reflect.Method.invokeNative(Native Method)
                        E      at java.lang.reflect.Method.invoke(Method.java:515)
                        E      at java.io.ObjectOutputStream.writeHierarchy(ObjectOutputStream.java:1053)
                        E      at java.io.ObjectOutputStrea
```

--- a/library/build.gradle
+++ b/library/build.gradle
@@ -2,6 +2,9 @@ subprojects {
 
   repositories {
     mavenCentral()
+//    maven {
+//      url 'https://oss.sonatype.org/content/repositories/snapshots/'
+//    }
   }
 
   buildscript {

--- a/library/leakcanary-analyzer/build.gradle
+++ b/library/leakcanary-analyzer/build.gradle
@@ -14,7 +14,7 @@ repositories {
 }
 
 dependencies {
-  compile 'com.squareup.haha:haha:1.2'
+  compile 'com.squareup.haha:haha:1.3'
   compile project(':leakcanary-watcher')
   testCompile 'junit:junit:4.12'
 }

